### PR TITLE
v2d & aabbox3d<f32> cleanups

### PR DESCRIPTION
**Pull request by nerzhul**
This replace many aabbox<f32> with the typedef we set.
Also move v2d header from tile.h to tile.cpp

### CODE REVIEW

File: src/clientmap.h [hunk 1]
```diff
@@ -115,7 +115,7 @@ class ClientMap : public Map, public scene::ISceneNode
 		renderMap(driver, SceneManager->getSceneNodeRenderPass());
 	}
 	
-	virtual const core::aabbox3d<f32>& getBoundingBox() const
+	virtual const aabb3f& getBoundingBox() const
```
Comment (est31) [inline src/clientmap.h]: Put the reference `&` next to the function name, to fix coding style

File: src/clientobject.h [hunk 2]
```diff
@@ -56,7 +56,7 @@ class ClientActiveObject : public ActiveObject
 	virtual void updateLight(u8 light_at_pos){}
 	virtual void updateLightNoCheck(u8 light_at_pos){}
 	virtual v3s16 getLightPosition(){return v3s16(0,0,0);}
-	virtual core::aabbox3d<f32>* getSelectionBox(){return NULL;}
+	virtual aabb3f* getSelectionBox(){return NULL;}
```
Comment (est31) [inline src/clientobject.h]: same here

File: src/clouds.h [hunk 3]
```diff
@@ -53,7 +53,7 @@ class Clouds : public scene::ISceneNode
 
 	virtual void render();
 	
-	virtual const core::aabbox3d<f32>& getBoundingBox() const
+	virtual const aabb3f& getBoundingBox() const
```
Comment (est31) [inline src/clouds.h]: and here

File: src/content_cao.cpp [hunk 4]
```diff
@@ -309,7 +309,7 @@ class ItemCAO : public ClientActiveObject
 
 	void initialize(const std::string &data);
 
-	core::aabbox3d<f32>* getSelectionBox()
+	aabb3f* getSelectionBox()
```
Comment (est31) [inline src/content_cao.cpp]: here too

File: src/content_cao.cpp [hunk 5]
```diff
@@ -674,7 +674,7 @@ GenericCAO::~GenericCAO()
 	removeFromScene(true);
 }
 
-core::aabbox3d<f32>* GenericCAO::getSelectionBox()
+aabb3f* GenericCAO::getSelectionBox()
```
Comment (est31) [inline src/content_cao.cpp]: and here

File: src/content_cao.h [hunk 6]
```diff
@@ -127,7 +127,7 @@ class GenericCAO : public ClientActiveObject
 
 	bool collideWithObjects();
 
-	core::aabbox3d<f32>* getSelectionBox();
+	aabb3f* getSelectionBox();
```
Comment (est31) [inline src/content_cao.h]: and here

File: src/particles.h [hunk 7]
```diff
@@ -52,7 +52,7 @@ class Particle : public scene::ISceneNode
 	);
 	~Particle();
 
-	virtual const core::aabbox3d<f32>& getBoundingBox() const
+	virtual const aabb3f& getBoundingBox() const
```
Comment (est31) [inline src/particles.h]: and here

**Comment by Ekdohibs** [comment]
Looks good!

File: src/sky.cpp [hunk 8]
```diff
@@ -94,7 +94,7 @@ void Sky::OnRegisterSceneNode()
 	scene::ISceneNode::OnRegisterSceneNode();
 }
 
-const core::aabbox3d<f32>& Sky::getBoundingBox() const
+const aabb3f& Sky::getBoundingBox() const
```
Comment (est31) [inline src/sky.cpp]: here as well

File: src/sky.h [hunk 9]
```diff
@@ -42,7 +42,7 @@ class Sky : public scene::ISceneNode
 	//! renders the node.
 	virtual void render();
 
-	virtual const core::aabbox3d<f32>& getBoundingBox() const;
+	virtual const aabb3f& getBoundingBox() const;
```
Comment (est31) [inline src/sky.h]: and here

File: src/wieldmesh.h [hunk 10]
```diff
@@ -53,7 +53,7 @@ class WieldMeshSceneNode: public scene::ISceneNode
 
 	virtual void render();
 
-	virtual const core::aabbox3d<f32>& getBoundingBox() const
+	virtual const aabb3f& getBoundingBox() const
```
Comment (est31) [inline src/wieldmesh.h]: here too

**Comment by est31** [comment]
Other than that: :+1:

**Comment by nerzhul** [comment]
rmlrmlr i only do a sed :p

**Comment by est31** [comment]
Well its only 11 places, that's no big deal to fix, is it

**Comment by nerzhul** [comment]
yeah :D you spam my mailbox :(

**Comment by est31** [comment]
Merged fefa148d694364cb4c60fbb4ffdc0f8ae6388df6

yet another item in your inbox :p

**Pull request closed without merging**